lib/http.js

```diff
@@ -0,0 +1,180 @@
+/*!
+ * Copyright (c) 2018-2019 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const asyncHandler = require('express-async-handler');
+const bedrock = require('bedrock');
+const brPassport = require('bedrock-passport');
+const {config} = bedrock;
+const storage = require('./storage');
+const {validate} = require('bedrock-validation');
+const uuid = require('uuid-random');
+require('bedrock-express');
+const {
+  ensureAuthenticated
+} = brPassport;
+const {BedrockError} = bedrock.util;
+
+// load config defaults
+require('./config');
+
+// module API
+const api = {};
+module.exports = api;
+
+bedrock.events.on('bedrock-express.configure.routes', app => {
+  const {baseUri} = config.server;
+  const cfg = config['data-hub-storage'];
+  const {routes} = cfg;
+  const baseStorageUrl = `${baseUri}${routes.dataHubs}`;
+
+  // create a new data hub
+  app.post(
+    routes.dataHubs,
+    ensureAuthenticated,
+    validate('bedrock-data-hub-storage.config'),
+    asyncHandler(async (req, res) => {
+      const {actor} = (req.user || {});
```

Comment:
Is this supposed to be `req.controller`, instead of `req.user`?
(From the sending end, bedrock-web-data-hub sends `controller` param, I don't see user anywhere).

Reply:
Ah, ignore that comment, looks like `user` and `controller` are different.